documentation/doxygen/extract_docstrings.py

```diff
@@ -64,6 +64,6 @@ def run_fast_scandir(dir, ext):    # dir: str, ext: list
     module = ast.parse(file_contents)
     ds = ast.get_docstring(module)
     if ds is not None:
-        with open(pyz_file_path + '.pyzdoc', 'w') as pyz_doc_file:
+        with open(pyz_file_path + 'zdoc', 'w') as pyz_doc_file:
```

Comment:
I think this new concatenation deserves at least a comment to explain that the input file name ends in .py and we write the output as `.pyzdoc`.

Reply:
how do you put comments "inline"with python ? the equivalent of:
```
 int i = 1; // I is equal to one
```
in C++ ?